### ISSUE TITLE
Error in translation of notifications email languages

### ISSUE THREAD

**Issue by vaazua** [pluginsGLPI/formcreator#818]
Hello, I use GLPI 9.2 and the 2.6 plugins of formcreator.

![plugins](https://user-images.githubusercontent.com/23486343/32240867-82d0dd86-be3c-11e7-92cc-753bd2a940b3.png)

When sending a form the email sends the notification in English, I have already changed as Spanish language of Spain since the file is_ES is the most complete.

In spite of that, he keeps sending notification notices in English.

![](https://user-images.githubusercontent.com/23486343/32248462-c28508b4-be53-11e7-93bc-0e164e9062d9.png)

![](https://user-images.githubusercontent.com/23486343/32248829-eac11394-be54-11e7-9324-74db10680ad3.png)

![](https://user-images.githubusercontent.com/23486343/32248891-1d3037ce-be55-11e7-9217-26f914f1a973.png)

![](https://user-images.githubusercontent.com/23486343/32248994-691ffac0-be55-11e7-9a12-08b7bfed6e0f.png)

I have even moved all the language files, I have copied the files es_ES.po and es_ES.mo to us_GB.po / us_GB.mo and en_US.po / en_US.mo and they still give me the messages in English.

![](https://user-images.githubusercontent.com/23486343/32249264-5fd2e634-be56-11e7-848f-ba6134e70502.png)

**Comment by btry** [comment]
Hi

I think this is only a missing configuration on your GLPI. Actually the notifications are provided in english only, and it is up to you to customize the text (if you want) and provide translation in other languages.

Go here
![image](https://user-images.githubusercontent.com/14139801/32251593-f21b683c-be91-11e7-8c64-a8e9b85ffffe.png)

then add a new translation and fill the form.

![image](https://user-images.githubusercontent.com/14139801/32251677-56f30ecc-be92-11e7-83ae-190e64bb8be4.png)

**Comment by vaazua** [comment]
In this case, the notification is exclusive of add-ons. The language and message files are located in ../plugins/formcreator/locales.

Other messages come out correctly in Spanish.

That's why I think it's something exclusive to plugins.

![](https://user-images.githubusercontent.com/23486343/32252860-b0cba95e-be65-11e7-9d2a-f99976ffb57c.png)

**Comment by btry** [comment]
Hi

When you install the plugin it creates new notifications in GLPI. The administrator can configure these new notifications like any other one (see the screenshots I pasted earlier). 

The new notifications are created using the localized text you pointed, and this text is put in the default translation. I think that while the plugin was installed, the current language was english, then the default translations is in english.

When a notification is generated to a user, the .mo files are *not* used for  translation. The translations created in the UI as I shown above are used. You must tranaslate like I described to ensuer the plugin will generate notifications for spanish users.
